frontend/lib/src/theme/themeConfigs.ts

```diff
@@ -15,29 +15,60 @@
  */
 
 import { lightThemePrimitives, darkThemePrimitives } from "baseui"
+import isObject from "lodash/isObject"
+import merge from "lodash/merge"
 import { baseuiLightTheme, baseuiDarkTheme } from "./baseui"
 import emotionBaseTheme from "./emotionBaseTheme"
 import emotionLightTheme from "./emotionLightTheme"
 import emotionDarkTheme from "./emotionDarkTheme"
 import { ThemeConfig } from "./types"
 
+declare global {
+  interface Window {
+    __streamlit?: {
+      LIGHT_THEME: ThemeConfig
```

Comment:
`ThemeConfig` has a very large surface area with hundreds of internal properties that are in constant change. I am fine with the change if SiS knows this is an internal structure under constant change. But if this is making `ThemeConfig` a part of our public API, we probably need to have more discussions on that because it would be quite a large commitment.

Reply:
The alternative would be to define an additional interface that only contains the minimal properties they need for the dark mode theming. Or, can we move the Snowflake light/dark theme into Streamlit itself and just activate snowflake-dark or snowflake-light?

Reply:
I agree to be a bit more explicit with the intermediary interface.